core/src/main/java/io/aiven/klaw/dao/migration/MigrateData2x10x0.java

```diff
@@ -0,0 +1,64 @@
+package io.aiven.klaw.dao.migration;
+
+import io.aiven.klaw.config.ManageDatabase;
+import io.aiven.klaw.dao.KwProperties;
+import io.aiven.klaw.dao.KwTenants;
+import io.aiven.klaw.helpers.KwConstants;
+import io.aiven.klaw.helpers.db.rdbms.InsertDataJdbc;
+import io.aiven.klaw.helpers.db.rdbms.SelectDataJdbc;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+@DataMigration(version = "2.10.0", order = 5)
+@Slf4j
+@Configuration // Spring will automatically scan and instantiate this class for retrieval.
+public class MigrateData2x10x0 {
+
+  @Autowired private InsertDataJdbc insertDataJdbc;
+  @Autowired private SelectDataJdbc selectDataJdbc;
+
+  @Autowired private ManageDatabase manageDatabase;
+
+  public MigrateData2x10x0() {}
+
+  protected MigrateData2x10x0(
+      SelectDataJdbc selectDataJdbc, ManageDatabase manageDatabase, InsertDataJdbc insertDataJdbc) {
+    this.selectDataJdbc = selectDataJdbc;
+    this.manageDatabase = manageDatabase;
+    this.insertDataJdbc = insertDataJdbc;
+  }
+
+  @MigrationRunner()
+  public boolean migrate() {
+    log.info("Start to migrate 2.10.0 data. Add new user email content property ");
+
+    List<Integer> tenantIds =
+        selectDataJdbc.getTenants().stream().map(KwTenants::getTenantId).toList();
+
+    for (int tenantId : tenantIds) {
+      List<KwProperties> kwPropertiesList =
+          new ArrayList<>(selectDataJdbc.selectAllKwPropertiesPerTenant(tenantId));
+      String newUserV2Content = "klaw.mail.newuseradded.v2.content";
```

Comment:
Seems like there is a constant NEW_USER_ADDED_V2_KEY available for this in MailUtils. Use that ?